public/api-docs/swaggerDoc.json

```diff
@@ -447,9 +525,37 @@
                 "password": {
                     "type": "string"
                 },
-                "address": {
+                "residencAddress": {
+                    "type": "string",
+                    "description": "residence address of the user"
+                },
+                "department": {
+                    "type": "string",
+                    "description": "User department"
+                },
+                "lineManager": {
+                    "type": "string",
+                    "description": "line manager to the user"
+                },
+                "gender": {
+                    "type": "string",
+                    "description": "User gender"
+                },
+                "phoneNumber": {
+                    "type": "integer",
+                    "description": "Phone number of user"
+                },
+                "role": {
+                    "type": "string",
+                    "description": "User role"
+                },
+                "preferredLanguage": {
+                    "type": "string",
+                    "description": "User preferred language"
+                },
+                "preferredCurrency": {
                     "type": "string",
-                    "description": "Address of WeTravel User"
```

Comment:
Please revert this change, you shouldn't be removing other people's changes when adding your

Reply:
I actually replaced it with ```  "residencAddress": {
                    "type": "string",
                    "description": "residence address of the user"
                }, ``` I'll revert however.

Reply:
I actually replaced it with ```  "residencAddress": {
                    "type": "string",
                    "description": "residence address of the user"
                }, ``` I'll revert however.

Reply:
Okay then, that's fine